src/subdomains/generic/kyc/enums/kyc.enum.ts

```diff
@@ -62,6 +63,8 @@ export function getKycTypeIndex(stepType?: KycStepType): number {
 }
 
 export function getIdentificationType(companyId: string): KycIdentificationType | undefined {
+  if (Object.values(WebhookType).some((t) => t === companyId)) return KycIdentificationType.ONLINE_ID;
+
   switch (companyId) {
     case 'dfxautonew':
```

Comment:
Isn't it a bit weird to mix these things together?

Reply:
Yeah but else you would have a long case list with all webhook types 😅 🤔

Reply:
I would just add the following in `completeSumsubIdent`:
```ts
const identificationType =  result.webhook.type === WebhookType.VIDEO_IDENT_STATUS_CHANGED
   ? KycIdentificationType.VIDEO_ID
   : KycIdentificationType.ONLINE_ID;
```